config.yml

```diff
@@ -143,3 +143,39 @@ docker:
   masterdata_ownership_filename: "2023-06-18/2023-06-18_AI_RMI Bespoke_Company Data Products_masterdata_ownership_2022Q4.csv"
   masterdata_debt_filename: "2023-06-18/2023-06-18_AI_RMI Bespoke_Company Data Products_masterdata_debt_2022Q4.csv"
   ar_company_id__factset_entity_id_filename: "2023-02-15/2023-02-15_AI_RMI_Bespoke_Company_Data_Products_Company_ID_List_2022Q4.csv"
+
+
+2023Q4:
+  inherits: docker
+  masterdata_ownership_filename: "2024-02-14_AI_RMI Bespoke_Company Data Products_masterdata_ownership_2023Q4.csv"
+  masterdata_debt_filename: "2024-02-14_AI_RMI Bespoke_Company Data Products_masterdata_debt_2023Q4.csv"
+  ar_company_id__factset_entity_id_filename: "2024-02-14_AI_RMI_Bespoke_Company_Data_Products_Company_ID_List_2023Q4.csv"
+  factset_financial_data_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_financial_data.rds"
+  factset_entity_info_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_entity_info.rds"
+  factset_entity_financing_data_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_entity_financing_data.rds"
+  factset_fund_data_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_fund_data.rds"
+  factset_isin_to_fund_table_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_isin_to_fund_table.rds"
+  factset_iss_emissions_data_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_iss_emissions.rds"
+  factset_issue_code_bridge_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_issue_code_bridge.rds"
+  factset_industry_map_bridge_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_industry_map_bridge.rds"
+  factset_manual_pacta_sector_override_filename: "timestamp-20231231T000000Z_pulled-20240217T135833Z_factset_manual_sector_override.rds"
+  imf_quarter_timestamp: "2023-Q4"
+  pacta_financial_timestamp: "2023Q4"
+  market_share_target_reference_year: 2023
+  time_horizon: 5
+  scenario_sources_list: ["GECO2023", "ISF2023", "WEO2023"]
+  sector_list: ["Automotive", "Power", "Oil&Gas", "Coal"]
+  other_sector_list: ["Steel", "Aviation", "Cement"]
+  zero_emission_factor_techs: ["Electric", "HydroCap", "NuclearCap", "RenewablesCap"]
+  green_techs: ["FuelCell", "Electric", "Hybrid", "RenewablesCap", "HydroCap", "NuclearCap", "FuelCell_HDV", "Electric_HDV", "Hybrid_HDV"]
+  scenario_raw_data_to_include: ["geco_2023", "isf_2023", "weo_2023"]
+  tech_exclude: ["OtherCap", "OtherFF", "Coking Plant", "Sintering Plant", "Direct Or Smelting Reduction Plant", "Pelletizing Plant", "Grinding Plant", "Passenger / Freight"]
```

Comment:
none of these exist directly in the raw AI data for 2023
"Passenger" and "Freight" do exist separately in the raw AI data for 2023

Reply:
Who is the right person t tag for input on this? @Antoine-Lalechere @NickPACTA ?

Reply:
I think we/I have to dig into the code and really clearly wrap my head around where/how they're being used, and verify if they exist anywhere in the data that might hit that code. If not, then I think it would be best to remove these (from the 2023Q4 config anyway).

On the other side, it might be prudent to review the relevant list of technologies that *do* exist in 20223Q4 and consider if any of them should be added here. For that, I'm honestly unsure who would/should make that decision. Maybe Nick?